package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "gulp-util": "^3.0",
-    "node-sass": "^3.0.0",
```

Comment:
^3.0.0 will get the latest 3.x.x version when installing so there is no need for that PR
Just remove node_modules/gulp-sass and reinstall and you will get the latest one